README/CREDITS

```diff
@@ -912,6 +917,11 @@ N: Matevz Tadel
 E: Matevz.Tadel@cern.ch
 D: Several contributions to the 3-d graphics and GL interface, WebEVE
 
+N: Yuka Takahashi
+E: Yuka.Takahashi@cern.ch
+D: runtime c++ modules integration in the CMS experiment
```

Comment:
This does not read like a ROOT contribution. Can you rephrase this to clarify the ROOT aspect of it? Maybe as simple as "integration of ROOT runtime modules in the CMS experiment"?